packages/notifi-react-card/lib/context/NotifiSubscriptionContext.tsx

```diff
@@ -94,13 +96,15 @@ const hasKey = <K extends string>(
 ): obj is object & { [k in K]: unknown } => {
   return typeof obj === 'object' && obj !== null && key in obj;
 };
+const contextId = uuid();
```

Comment:
probably more useful if we do this inside the Provider render function as a `useMemo`